src/sections/Calendar/index.js

```diff
@@ -6,9 +6,39 @@ import interactionPlugin from "@fullcalendar/interaction";
 import googleCalendarPlugin from "@fullcalendar/google-calendar";
 import { Container } from "../../reusecore/Layout";
 import PageHeader from "../../reusecore/PageHeader";
+import Table from "../../components/MeetInfo-Table";
+import { meet_links_data } from "./meetLinksData";
 
 const Calendar = () => {
 
+    const columns = React.useMemo(
+        () => [
+            {
+                Header: "Meeting",
+                accessor: "meeting",
+            },
+            {
+                Header: "Slack Channel",
+                accessor: "slack_channel",
+            },
+            {
+                Header: "Meeting Minutes",
+                accessor: "meeting_minutes",
+            },
+            {
+                Header: "Meeting Link",
+                accessor: "meeting_link",
+            },
+            {
+                Header: "Meeting Recordings",
+                accessor: "meeting_recordings",
+            },
```

Comment:
@leecalcote you will have to add a New Header here and the `accessor: "day"` corresponding to it.

Reply:
Aha!

_yeah, I knew that. just seeing if you did._

Reply:
Should be a bit easier after https://github.com/layer5io/layer5/pull/1351#discussion_r554529435

Reply:
Good cover up @leecalcote 😆

Reply:
Someone has to keep you guys on your toes. 😆 

_Even if it means that I have to ask embarrassing questions._